Source/DafnyCore/Options/CommonOptionBag.cs

```diff
@@ -252,6 +252,10 @@ public enum GeneralTraitsOptions {
     @"In case the Dafny source code is translated to another language, emit that translation.") {
   };
 
+  public static readonly Option<bool> WarnAsErrors = new("--warn-as-errors", () => true, "(Deprecated). Please use --allow-warnings instead") {
```

Comment:
```suggestion
  public static readonly Option<bool> WarnAsErrors = new("--warn-as-errors", () => true, "(Deprecated). Please use --allow-warnings:false instead (which is the default)") {
```

Reply:
I wish GH would allow me to commit this suggestion without requiring re-approval.